docs/pyretriever.rst

```diff
@@ -189,24 +204,52 @@ A description of default parameters mentioned above:
 
 .. code-block:: python
 
-  compile   (Bool): Setting True recompiles scripts upon installation.
+  compile         (Bool): Setting True recompiles scripts upon installation.
+
+  database_name (String): Specify database name. For postgres, mysql users.
+
+  debug           (Bool): Setting True helps in debugging in case of errors.
 
-  database_name(String): Specify database name. For postgres, mysql users.
+  file          (String): Enter file_name for database. For msaccess, sqlite users.
 
-  debug     (Bool): Setting True helps in debugging in case of errors.
+  host          (String): Specify host name for database. For postgres, mysql users.
 
-  file      (String): Enter file_name for database. For msaccess, sqlite users.
+  password      (String): Specify password for database. For postgres, mysql users.
 
-  host      (String): Specify host name for database. For postgres, mysql users.
+  port             (Int): Specify the port number for installtion. For postgres, mysql users.
 
-  password  (String): Specify password for database. For postgres, mysql users.
+  quiet           (Bool): Setting True minimizes the console output.
 
-  port      (Int): Specify the port number for installtion. For postgres, mysql users.
+  table_name    (String): Specify the table name to install.
+
+  use_cache       (Bool): Setting False reinstall scripts if it is already installed.
+
+  user          (String): Specify the user_name. For postgres, mysql users.
+  
+Examples to Installing Datasets:
+
+Here, we are installing the CSV file to the dataset ``wine-composition`` to our current-working directory.
```

Comment:
Change to: 
Here, we are installing the dataset ``wine-composition`` as a CSV file in our current-working directory.